docs_source_files/content/support_packages/working_with_cookies.de.md

```diff
@@ -338,7 +473,29 @@ const {Builder} = require('selenium-webdriver');
 })();
   {{< / code-panel >}}
   {{< code-panel language="kotlin" >}}
-// We don't have a Kotlin code sample yet -  Help us out and raise a PR  
+import org.openqa.selenium.Cookie
+import org.openqa.selenium.chrome.ChromeDriver
+
+fun main() {
+    val driver = ChromeDriver()
+    try {
+        driver.get("https://example.com")
+        driver.manage().addCookie(Cookie("test1", "cookie1"));
+        val cookie1 = Cookie("test2", "cookie2")
+        driver.manage().addCookie(cookie1);
+
+        // delete a cookie with name 'test1'
+        driver.manage().deleteCookieNamed("test1");
+        /*
+         Selenium Java bindings also provides a way to delete
+         cookie by passing cookie object of current browsing context.
+         This feature also applies for kotlin as it uses selenium jar java bindings
+        */
```

Comment:
Is this comment really needed? I would imagine that if someone is using Kotlin for WebDriver is aware that they are using the Java bindings as well.

Reply:
hi @diemol , 

logically yes.  I've just placed it for reference if the user directly jumps to kotlin code references.

It may be bit useful if the user has no idea on java bindings and a direct kotlin user (rare case)